### PR TITLE
code comment and formatting updates to the pointing directory of MCP,…

### PR DESCRIPTION
**Pull request by ianlowe13**
… end of current backlogged directories

### CODE REVIEW

File: mcp/pointing/pointing.c [hunk 1]
```diff
@@ -667,6 +723,8 @@ static void record_gyro_history(int m_index, gyro_history_t *m_gyhist, gyro_read
 //   return(1);
 // }
 
+
+// deprecated, should be updated with new XSC stuff
```
Comment (shubhagrawal30) [inline mcp/pointing/pointing.c:727]: add a TODO tag.

File: mcp/pointing/pointing.c [hunk 2]
```diff
@@ -1070,6 +1134,8 @@ static void EvolveAzSolution(struct AzSolutionStruct *s, double ifroll_gy,
     s->since_last++;
 }
 
+
+// deprecated, replace with new XSC stuff
```
Comment (shubhagrawal30) [inline mcp/pointing/pointing.c:1138]: TODO tag

File: mcp/pointing/pointing.c [hunk 3]
```diff
@@ -1946,6 +2018,8 @@ void set_position(double m_lat, double m_lon)
     SIPData.GPSpos.lon = m_lon;
 }
 
+
+// deprecated, change to new XSC stuff
```
Comment (shubhagrawal30) [inline mcp/pointing/pointing.c:2022]: TODO tag

File: mcp/pointing/pointing.c [hunk 4]
```diff
@@ -188,6 +237,7 @@ void SetSafeDAz(double ref, double *A)
   }
 }
 
+
 /**
  * @brief Use the world magnetic model, atan2 and a lookup table to convert
  * mag_x and mag_y to mag_az.
```
Comment (shubhagrawal30) [inline mcp/pointing/pointing.c:243]: @return --> @param

File: mcp/pointing/time_lst.c [hunk 5]
```diff
@@ -165,16 +185,18 @@ tt_pred_t tt_predictions[] =
         { 2458757.5, 71. }
     };
 
+
+// UT1 is "universal time" UTC is "coordinated universal time" and is linked to atomic clocks.
+// Sometimes we get a leap second added to fix UTC to UT1 which varies due to Earth's rotation.
 /**
- * Computes the approximate difference between UT1 and UTC.  This is on the order
+ * @brief Computes the approximate difference between UT1 and UTC.  This is on the order
```
Comment (shubhagrawal30) [inline mcp/pointing/time_lst.c:192]: does not seem like we use this function anywhere in the repo anyway...

File: mcp/pointing/angles.c [hunk 6]
```diff
@@ -54,6 +80,15 @@ double wrap_to(double angle, double max)
     return angle;
 }
 
+
+/**
+ * @brief discretizing angles to integer values, this wraps
+ * the angle to the range of [0, max]
```
Comment (shubhagrawal30) [inline mcp/pointing/angles.c:86]: a bit confusing docstring, could be interpreted as "this function takes in a doublwe and rounds it off"

File: mcp/pointing/pointing.c [hunk 7]
```diff
@@ -159,20 +189,39 @@ static double sun_az, sun_el; // set in SSConvert and used in UnwindDiff
 
 #define M2DV(x) ((x / 60.0) * (x / 60.0)) // used to convert gyro spec std dev to a variance in deg
 
-// limit to 0 to 360.0
+
+/**
+ * @brief limit to 0 to 360.0
```
Comment (shubhagrawal30) [inline mcp/pointing/pointing.c:194]:
```suggestion
 * @brief limit to 0 to 360.0 degrees
```

File: mcp/pointing/pointing.c [hunk 8]
```diff
@@ -159,20 +189,39 @@ static double sun_az, sun_el; // set in SSConvert and used in UnwindDiff
...
 void NormalizeAngle(double *A)
 {
   *A = fmod(*A, 360.0);
   if (*A < 0)
     *A += 360.0;
 }
 
+
+/**
+ * @brief adjust *A to be within +-180 of ref
```
Comment (shubhagrawal30) [inline mcp/pointing/pointing.c:207]: add units

File: mcp/pointing/angles.c (hunk 6 above)
Comment (ianlowe13) [reply mcp/pointing/angles.c:86]: fair, trying to fix

**Review by evanmayer — APPROVED** [review]

**Pull request merged**